pyomo/future.py

```diff
@@ -28,14 +35,30 @@
 
 """
 
+solver_factory_v1: "_solvers.SolverFactoryClass"
+solver_factory_v2: "_appsi.SolverFactoryClass"
+solver_factory_v3: "_contrib.SolverFactoryClass"
+
```

Comment:
We need to delete these lines so that the `from pyomo.__future__ import solver_factory_v3` logic works.

(And some time I should add unitttests for this [maybe put it in `pyomo/environ/tests/test_future.py`?])

Reply:
Doh.  You're right, of course.